<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDailyTaskWindow</class>
 <widget class="QWidget" name="AddDailyTaskWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="taskEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pointsEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="amountEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="taskLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>58</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Task</string>
   </property>
  </widget>
  <widget class="QLabel" name="pointsLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>58</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Points</string>
   </property>
  </widget>
  <widget class="QLabel" name="amountLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>58</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>83</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>58</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
